packages/create-astro/src/messages.ts

```diff
@@ -60,10 +60,10 @@ export const getName = () =>
 		});
 	});
 
-export const getVersion = (packageManager: string, packageName: string, fallback = '') =>
+export const getVersion = (packageManager: string, packageName: string, ref: string, fallback = '') =>
 	new Promise<string>(async (resolve) => {
 		let registry = await getRegistry(packageManager);
-		const { version } = await fetch(`${registry}/${packageName}/latest`, {
+		const { version } = await fetch(`${registry}/${packageName}/${ref}`, {
```

Comment:
as we've discovered during the offsite, this ends up not working. In one time ref refers to a branch, in the other it refers to a npm tag. For instance, `astro@next` does not exist, so setting `--ref next` would not work.